query.go

```diff
@@ -1096,6 +1098,18 @@ func (s *ChainService) sendTransaction(tx *wire.MsgTx, options ...QueryOption) e
 					response, sp.Addr(),
 				)
 				rejections[*broadcastErr]++
+
+				// To ever get (numReplied == len(rejections)), it is necessary
+				// to also increment numReplied. However, this would be a
+				// breaking change since common innocuous errors such as Mempool
+				// and Confirmed do not presently cause a non-nil error return.
```

Comment:
nit: wrap at 80 chars

Reply:
This commit is removed, so it's a moot point, but I think we must have our editors configured differently or something.  My comment wrapping extension puts it like this, and I am seeing it come right to the limit but not over.  My cursor at the beginning of the line says 1, and at the end of "Mempool" says 81.  Possibly we have different conventions?  :shrug:  Only pressing the nit so I can adhere to your conventions in the future...

Reply:
are you sure your IDE sets tab size to 8 chars?

Reply:
Oh, tabs. Noi use 4.  That explains that